### PR TITLE
Adding SituationFullRef to LIR and SER

### PR DESCRIPTION
**Pull request by ue71603**
Adding SituationRefs to the last two services
* LocationInformation /PlaceInformation
* StopEvent

Had to place it a bit strangly for Place, but I guess it works as well.
SER already had SituationRefs for the Calls. I thought it would be good to have it on the StopEvent itself as well.

Fixes: https://github.com/VDVde/OJP/issues/260

### CODE REVIEW

File: OJP/OJP_PlaceSupport.xsd [hunk 1]
```diff
@@ -430,6 +430,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="GeoPosition" type="siri:LocationStructure"/>
+			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded">
```
Comment (Aurige) [inline OJP/OJP_PlaceSupport.xsd:433]: Probably add a wrapper since it is unbounded ...

Comment (ue71603) [reply OJP/OJP_PlaceSupport.xsd]: wrappers done

**Review by AndreasAtSBB — DISMISSED** [review]
Changes are OK.

**Review by trurlurl — DISMISSED** [review]

**Review by skinkie — DISMISSED** [review]

File: OJP/OJP_SituationSupport.xsd [hunk 2]
```diff
@@ -31,4 +31,12 @@
 			<xs:extension base="siri:SituationFullRefStructure"/>
 		</xs:complexContent>
 	</xs:complexType>
+	<xs:complexType name="SituationRefList">
+		<xs:annotation>
+			<xs:documentation>List of references to SITUATIONs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (sgrossberndt) [inline OJP/OJP_SituationSupport.xsd:39]: If such a optional list (due to `<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0">`) is given it should not be empty, correct? So I think it should be `minOccurs="1"` (which is the default, therefore the `minOccurs` should be omitted)

File: OJP/OJP_JourneySupport.xsd [hunk 3]
```diff
@@ -231,7 +231,7 @@
 					<xs:documentation>Indicates whether this service needs some kind of reservation or booking to run.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded"/>
+			<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0"/>
```
Comment (sgrossberndt) [inline OJP/OJP_JourneySupport.xsd:234]: annotation documentation is missing on all these `SituationFullRefs`.

**Review by sgrossberndt — CHANGES_REQUESTED** [review]
In general: Please have a look at the generated docs/generated/index.html using https://htmlpreview.github.io/ for each change to make sure everything is working for each change.

File: OJP/OJP_SituationSupport.xsd (hunk 2 above)
Comment (ue71603) [reply OJP/OJP_SituationSupport.xsd]: changed

File: OJP/OJP_JourneySupport.xsd (hunk 3 above)
Comment (ue71603) [reply OJP/OJP_JourneySupport.xsd]: changed

**Review by skinkie — DISMISSED** [review]

**Review by AndreasAtSBB — APPROVED** [review]
Change:
- ref=SituationFullRef ...
- name=situationFullRefs ....

**Review by skinkie — DISMISSED** [review]

**Review by skinkie — DISMISSED** [review]

**Review by trurlurl — DISMISSED** [review]

**Review by skinkie — APPROVED** [review]
@trurlurl you may approve again...

**Review by AndreasAtSBB — APPROVED** [review]
Now always:
<xs:element name="SituationFullRefs" type="SituationRefList" minOccurs="0">
				<xs:annotation>
					<xs:documentation>A list of references to SITUATIONs.</xs:documentation>
				</xs:annotation>
			</xs:element>

and HTML doc updated.

**Pull request merged**